draft-ietf-quic-transport.md

```diff
@@ -921,6 +924,30 @@ explained in more detail as they are referenced later in the document.
 All QUIC frames are idempotent.  That is, a valid frame does not cause
 undesirable side effects or errors when received more than once.
 
+The Frame Type field uses a variable length integer encoding (see
+{{integer-encoding}}) with one exception.  To ensure simple and efficient
+implementations of frame parsing, a frame type MUST use the shortest possible
+encoding.  Though a two-, four- or eight-octet encoding of the frame types
+defined in this document is possible, the Frame Type field for these frames are
+encoded on a single octet.  For instance, though 0x4007 is a legitimate
+two-octet encoding for a variable-length integer with a value of 7, PING frames
+are always encoded as a single octet with the value 0x07.
```

Comment:
And what am I required to do if I encounter a 2+ byte encoding of 7?

Reply:
Agreed, I think we either need to say you MAY close the connection or this is a SHOULD not a MUST